crates/binstalk/src/ops/resolve.rs

```diff
@@ -465,6 +446,7 @@ impl PackageInfo {
         } else {
             Ok(Some(Self {
                 overrides: mem::take(&mut meta.overrides),
+                signing: mem::take(&mut meta.signing),
```

Comment:
I think this is the reason why signing test is failing, because you take it here so it's set to None.

I don't think signing here is ever used, so it can be removed.

Reply:
Ahhh thank you